### PR TITLE
[PWGHF] Use the Hf event selection for both tracks and charm hadrons

### PR DESCRIPTION
**Pull request by zhangbiao-phy**
- Use the Hf event selection in the producer task
- Update the default ccdb path of the software trigger

### CODE REVIEW

**Review by fgrosa — COMMENTED** [review]
It looks good to me @zhangbiao-phy, thanks! You can set it as ready for review, I will approve once the tests are green

**Comment by mfaggin** [comment]
@zhangbiao-phy there is this PR https://github.com/AliceO2Group/O2Physics/pull/8700 going to be merged, can you rebase on it once merged? Thanks.

**Comment by zhangbiao-phy** [comment]
> there is this PR #8700 going to be merged, can you rebase on it once merged? Thanks.

Hi @mfaggin, Thanks for reminding me about this! sure, l will wait it and rebase.

**Review by fgrosa — APPROVED** [review]

**Pull request merged**
